config/initializers/statsd.rb

```diff
@@ -183,3 +182,13 @@
 
   StatsD.measure('facilities.lighthouse', duration, tags: ['facilities.lighthouse'])
 end
+
+require 'active_support/notifications'
+
+ActiveSupport::Notifications.subscribe('request.faraday') do |name, start_time, end_time, _, env|
+  url = env[:url]
+  duration = end_time - start_time
+  http_method = env[:method].to_s.upcase
+  StatsD.measure('api.external.request.duration', duration, tags: ["host:#{url.host}"])
```

Comment:
this metric is pretty much a duplicate of job:breakers_http_request_duration:sum, that's displayed on http://grafana.vfs.va.gov/d/000000050/backend-service-report?orgId=1 I'd think. t

Reply:
Yes but this would actually automatically instrument any `faraday` requests with this detail rather than have to follow the facilities pattern as seen in lines above 185